app/views/state_file/questions/eligible/edit.html.erb

```diff
@@ -33,7 +33,7 @@
       <% end %>
       <div class="with-top-separator">
         <p class="text--bold"><%=t('.want_to_claim_heading') %></p>
-        <p><%=t('.want_to_claim_vita') %></p>
+        <p><%=t('.want_to_claim_vita', state_name: States.name_for_key(params[:us_state].upcase)) %></p>
```

Comment:
The translation file already the right thing:

https://github.com/codeforamerica/vita-min/blob/1d2d3ac42b0d4eecfc7b5d16290df1a3242a471d/config/locales/en.yml#L2178